docs/sources/getting-started/timeseries-dimensions.md

```diff
@@ -0,0 +1,85 @@
++++
+title = "Time series dimensions"
+description = "time series dimensions"
+keywords = ["grafana", "intro", "guide", "concepts", "timeseries", "labels"]
+type = "docs"
+aliases = ["/docs/grafana/latest/guides/timeseries-dimensions"]
+[menu.docs]
+name = "Time series dimensions"
+identifier = "time_series_dimensions"
+parent = "guides"
+weight = 300
++++
+
+# Time series dimensions
+
+In at the end of the ["Time series databases" section of "Introduction to time series"]({{< relref "timeseries.md#time-series-databases" >}}) the concept of Labels (a.k.a tags) is introduced:
+
+> Another feature of a TSDB is the ability to filter measurements using _tags_. Each data point is labeled with a tag that adds context information, such as where the measurement was taken. ...
+
+With time series data, the data often contains more than a single series, and is a set of multiple time series. Many Grafana data sources support this type of data.
+
+{{< docs-imagebox img="/img/docs/example_graph_multi_dim.png" class="docs-image--no-shadow" max-width="850px" >}}
+
+The common case is issuing a single query for a measurement with one or more additional properties as dimensions. For example querying a temperature measurement along with a location property as a dimension of the measurement. In this case, multiple series are returned back from that single query. Each series in this case has unique location. To uniquely identify these series within a set of time series, Grafana stores this information in _Labels_.
```

Comment:
```suggestion
The common case is issuing a single query for a measurement with one or more additional properties as dimensions. For example, querying a temperature measurement along with a location property as a dimension of the measurement. In this case, multiple series are returned back from that single query. Each series in this case has unique location. To uniquely identify these series within a set of time series, Grafana stores this information in _labels_.
```

Reply:
When writing for the web, avoid making sentences that are more than 25 words long or paragraphs more than three sentences long. Please break this paragraph into smaller ones.

For the example, consider adding a markdown table to illustrate your point.